CHANGES.md

```diff
@@ -13,6 +13,7 @@ Unreleased changes refer to our current [master branch](https://github.com/openc
 * Fixed `Too many connections to ldap` when openchange runs on samba as member of a domain.
 * Fixed `Mark All as Read` feature regression bug introduced by 7737bdf6
 * Address book working much better than before
+* ResolveNames and ResoveNamesW operations perform a better Ambiguous Name Resolution search, which returns more matches.
```

Comment:
mmm.. maybe to have both an 'ADDED' and a 'IMPROVEMENTS' section is confusing. What about merging the two in 'IMPROVEMENTS'?